config/locales/en.models.yml

```diff
@@ -76,6 +76,8 @@ en:
             base:
               conflict: The reservation conflicts with another reservation.
               conflict_in_cart: The reservation conflicts with another reservation in your cart. Please purchase or remove it then continue.
+            reserve_start_at:
+              after_cutoff: must be before the cutoff time for this instrument
```

Comment:
I think it might be nice to include the # of hours in this message. So something like "must be at least %{hours} hours in the future".